app/services/foreman_gutterball/gutterball_service.rb

```diff
@@ -26,26 +26,12 @@ class GutterballService < GutterballResource
     def initialize(_options = {})
     end
 
-    def available_reports
+    def reports(report_key = nil, query_params = nil)
```

Comment:
sure, we could just proxy all the params straight through to gutterball, but candlepin/gutterball uses different terminology referring to entities than katello does (i.e. - katello users would have no idea what a consumer is). it would be a **slight** pain, but it'll allow us to have params that make more sense to katello users.